1-data-structures/1-java/4-quick sort

```diff
@@ -0,0 +1,43 @@
+Quick sort is a popular sorting algorithm in computer science that uses a divide-and-conquer approach to sort a list of elements. It works by selecting a "pivot" element from the list and then partitioning the other elements into two sub-arrays according to whether they are less than or greater than the pivot. This process is then applied recursively to the sub-arrays until the entire list is sorted.
```

Comment:
This file not like readme file you have to change the file name with Quick-sort.MD

Reply:
I have made the change